Commands/security/api-collection/readme.md

```diff
@@ -0,0 +1,17 @@
+# [Group] _security api-collection_
+
+Manage Azure API Management API connections to Microsoft Defender for APIs.
```

Comment:
"Discover, manage, and view security insights for API collections"

Reply:
Updated